index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Courgette|Special+Elite|Arvo' rel='stylesheet' type='text/css'>
+
+    <title>Where in the World Quiz</title>
+</head>
+
+<body>
+
+    <header>
+        <h1>Where in the World are You?</h1>
+    </header>
+
+    <section class="introduction">
+        <h2 class="top-rule">Can you figure out where in the world each photo was taken?</h2>
+        <h2>Study the photos and look for hints!</h2>
+        <button id="start" class="startQuiz">Start Jetsetting!</button>
+    </section>
+
+    <section class="questions">
+        <div id="question-wrapper">
+            <h2 id="addQuestionText"></h2>
+        </div>
+        <div id="image"><img src="images/Montmartre.jpg" />
+        </div>
+        <div class="choices-list">
+            <ul id="userChoices" class="choices">
+
+
+            </ul>
+        </div>
+        <div class="button-placer">
+            <button id="submit" class="guess">Guess</button>
+        </div>
+
+        <h3 id="question-counter" class="question-tracker" id="count"></h3>
```

Comment:
Right because it is the last one read....ok got it!

On Wed, Jul 27, 2016 at 11:43 AM Aric Johnson <notifications@github.com>
wrote:

> In index.html
> <https://github.com/aric87/quiz_prtest/pull/1#discussion_r72475976>:
>
> > +        <div id="question-wrapper">
> > +            <h2 id="addQuestionText"></h2>
> > +        </div>
> > +        <div id="image"><img src="images/Montmartre.jpg" />
> > +        </div>
> > +        <div class="choices-list">
> > +            <ul id="userChoices" class="choices">
> > +
> > +
> > +            </ul>
> > +        </div>
> > +        <div class="button-placer">
> > +            <button id="submit" class="guess">Guess</button>
> > +        </div>
> > +
> > +        <h3 id="question-counter" class="question-tracker" id="count"></h3>
>
> very likely it would be the second because the first one would be set,
> then the second one would overwrite it. It's possible that *some*
> browsers might recognize both, but likely not all
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/aric87/quiz_prtest/pull/1/files/89f75e452744e07cbf3548ef74a7a6052a27555a#r72475976>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARHJXNPZ_TlUUyNqa3GgllNKTgAZZVYiks5qZ4qegaJpZM4I1uZa>
> .
>